Classifiers/classifier-EWS_v2.json

```diff
@@ -0,0 +1,297 @@
+{
+	"id": "c9332555-348b-4d91-85b4-e345cdba12c8",
+	"version": -1,
+	"modified": "2018-04-08T19:42:30.856069216Z",
+	"defaultIncidentType": "",
```

Comment:
Shouldn't be `"Phishing"`?

Reply:
Where do you set this up in UI?  haven't seen that

Reply:
incident type in the instance settings:
![image](https://user-images.githubusercontent.com/10514677/38562952-6911203c-3ce4-11e8-913d-b919d9c6c4fb.png)

and than just put an empty classification key

(a bad flow, we have an issue for that - https://github.com/demisto/etc/issues/11403)

Reply:
very